learningresources/tests/test_api.py

```diff
@@ -157,36 +152,6 @@ def test_invalid_create_repo(self):
         self.assertIn("description", ex.exception.args[0])
 
 
-class TestStaticAsset(LoreTestCase):
```

Comment:
It was causing an error in `tearDown` that I spent too much time trying to figure out (no errors in the test itself), and it wasn't needed for coverage due to the new tests I added.